quixstreams/platforms/quix/api.py

```diff
@@ -32,7 +32,7 @@ def __init__(
         api_version: Optional[str] = None,
         default_workspace_id: Optional[str] = None,
     ):
-        self._portal_api = portal_api or QUIX_ENVIRONMENT.portal_api
+        self._portal_api = portal_api or QUIX_ENVIRONMENT.portal_api or "https://portal-api.platform.quix.io/"
```

Comment:
One suggestion - let's move the URL to the module-level variable, e.g. `DEFAULT_PORTAL_API_URL`, to better explain what this URL means.